### PR TITLE
[ft-#162414124] create component to write article

### PR DESCRIPTION
**Pull request by Luckzman**
#### What does this PR do?
Implement user authentication using social media

#### Description of Task to be completed?
Implement component to write articles.

#### How should this be manually tested?
pull the repository and cd into the branch and then run npm test

#### What are the relevant pivotal tracker stories?
PT link - https://www.pivotaltracker.com/story/show/162414124

#### Screenshots
createarticle

### CODE REVIEW

File: src/components/Editor/Editor.scss [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  margin-top: 2rem;
+  .item {
+    padding: 1em 2em;
+    margin: 0;
+    color: $profileIconColor;
+    text-transform: uppercase;
+    border-radius: 0;
+  }
+  .active.item {
+    -webkit-box-shadow: none;
```
Comment (iAfolayan) [inline src/components/Editor/Editor.scss]: Nice job 
kindly make use of auto-prefixer.

Comment (Luckzman) [reply src/components/Editor/Editor.scss]: Thanks, I have fixed it

**Review by iAfolayan — APPROVED** [review]

**Review by codrex — APPROVED** [review]

File: src/components/tags/test/__snapshots__/tags.test.js.snap [hunk 2]
```diff
@@ -5,86 +5,44 @@ exports[`button test should test for the button component 1`] = `
   className="category"
 >
   <ListItem
-    key="Politics"
+    key="[object Object]"
```
Comment (codrex) [inline src/components/tags/test/__snapshots__/tags.test.js.snap:8]: `key="[object Object]"` Keys are meant to be unique identifiers of a component. 
` #[object Object]` this component will render an object to the view. 

Please revert the changes you made to the previous state.

File: __mocks__/index.js [hunk 3]
```diff
@@ -94,5 +94,5 @@ export {
   ARTICLE_AUTHOR,
   ARTICLE_DATE,
   ARTICLE_READ_TIME,
-  ARTICLE_CONTENT,
+  ARTICLE_CONTENT
```
Comment (codrex) [inline __mocks__/index.js:97]: This folder is meant for store jest runtime dependency and not application mock data. Please make use of the mock Data folder.

Comment (Luckzman) [reply __mocks__/index.js:97]: This work was done by Ibrahim Alausa, moving this file might break the code base, I think Alausa will be the best person to move the file.

File: src/components/tags/test/__snapshots__/tags.test.js.snap (hunk 2 above)
Comment (Luckzman) [reply src/components/tags/test/__snapshots__/tags.test.js.snap:8]: fixed
